security/Authentication.rst

```diff
@@ -218,7 +218,7 @@ the only one able to login to Kylo.  Obviously, this profile should only be used
 +--------------------------------+----------+---------------+-----------------------------------+
 | Property                       | Required | Example Value | Description                       |
 +================================+==========+===============+===================================+
-| authenticationService.username | Yes      | ``dlamin``    | The username of the administrator |
+| authenticationService.username | Yes      | ``dladmin``    | The username of the administrator |
```

Comment:
Well this is embarrassing. Thanks for catching this! Could you fix the extra spacing so bars | line up?